docs/admin/admission-controllers.md

```diff
@@ -252,6 +252,11 @@ This plugin ignores any `PersistentVolumeClaim` updates, it acts only on creatio
 See [persistent volume](/docs/user-guide/persistent-volumes) documentation about persistent volume claims and
 storage classes and how to mark a storage class as default.
 
+### DefaultTolerationSeconds
+
+This plug-in set default forgiveness toleration for pods (that has no forgiveness tolerations) to tolerate taints
```

Comment:
### Suggestion: 
This plug-in sets the default forgiveness toleration for pods, which have no forgiveness tolerations, to tolerate the taints `notready:NoExecute` and `unreachable:NoExecute` for 5 minutes.